packages/react-reconciler/src/ReactHookEffectTags.js

```diff
@@ -18,3 +18,32 @@ export const HasEffect = /* */ 0b0001;
 export const Insertion = /* */ 0b0010;
 export const Layout = /*    */ 0b0100;
 export const Passive = /*   */ 0b1000;
+
+export const listFlags = (inputFlag: HookFlags): string[] => {
```

Comment:
Thoughts on adding these? I found them handy in debugging. When printing a flag, instead of seeing a number, you get a list of strings of all the flags that have been OR'd together.